cea/optimization/slave/daily_storage/load_leveling.py

```diff
@@ -26,16 +26,16 @@ def storage_temperature(self, Q_storage_possible_W, activation):
         T_tank_C = self.T_tank_K - 273
         T_ground_C = self.T_ground_average_K - 273
         if activation == "charge":
-            Qc_from_Tank_W = 0.0
+            Qc_from_tank_W = 0.0
```

Comment:
Not sure how this fits with the overall CEA nomenclature – is this total cooling demand? In that case, following the CEA demand results file, it should probably be called "QC_from_tank_W". No big deal though.

Reply:
This refers to the cooling demand that can be met by the PCM storage. Since the storage tank currently gets activated before every other cooling technology this is in theory the total cooling demand of the DCN (or the current charge of the storage, whichever is smaller).